community/blog/2022-09 New types of request tracking in Arcus Observability v2.6/New types of request tracking in Arcus Observability v2.6.md

```diff
@@ -0,0 +1,93 @@
+# New types of request tracking in Arcus Observability v2.6
+The Arcus Observability v2.6 release contains all sorts of new ways to track requests in Application Insights. Let's have a look!
+
+Previously, we only supported HTTP and Azure Service Bus request tracking in our telemetry extensions. The v2.6 expands on this with great new and extensible ways to track telemetry.
+
+## Azure EventHubs request tracking
+We are working on fully supporting Azure EventHubs in our [Messaging library](https://messaging.arcus-azure.net/). This means that we should be able to track EventHubs requests, as the messaging system will retrieve and process these events. When we update our Messaging library, events that gets processed by your implemented message handlers will automatically be tracked.
+
+```csharp
+using Microsoft.Extensions.Logging;
+
+ILogger logger = ...
+
+bool isSuccessful = false;
+using (var measurement = DurationMeasurement.Start())
+{
+    try
+    {
+        // Process event...
+    }
+    finally
+    {
+        logger.LogEventHubsRequest("<my-eventhubs-namespace>", "<my-eventhubs-name>", isSuccessful, measurement);
+        // Output: Azure EventHubs from Process completed in 0.00:12:20.8290760 at 2021-10-26T05:36:03.6067975 +02:00 - (IsSuccessful: True, Context: {[EventHubs-Namespace, <my-eventhubs-namespace>]; [EventHubs-Name, <my-eventhubs-name>]; [EventHubs-ConsumerGroup, $Default]; [TelemetryType, Request]})
+    }
+}
+```
+
+We also provide overloads that take in an additional Azure EventHubs consumer group and an operation name which provides a functional description of the event.
+
+## Azure Functions (isolated) HTTP trigger request tracking
+With the arrival of isolated Azure Functions, processing HTTP requests has changed. Previously, we were able to re-use the ASP.NET Core `HttpRequest`/`IActionResult` but the isolated HTTP trigger variant stepped away from the reusable approach and uses new types: `HttpRequestData`/`HttpResponseData`. This impact how we track events with our Observability library as there is no default conversion available between the two approaches.
```

Comment:
```suggestion
With the arrival of isolated Azure Functions, processing HTTP requests has changed. Previously, we were able to re-use the ASP.NET Core `HttpRequest`/`IActionResult` but the isolated HTTP trigger variant stepped away from the reusable approach and uses new types: `HttpRequestData`/`HttpResponseData`. This has an impact on how we track events with our Observability library as there is no default conversion available between the two approaches.
```

"This impacts " or "This has an impact on "

Reply:
Aha, yeah, let's go with 'This impacts' as it's shorter.